src/cljdoc/server/sitemap.clj

```diff
@@ -0,0 +1,30 @@
+(ns cljdoc.server.sitemap
+  (:require [sitemap.core :refer [generate-sitemap]]
+            [clojure.tools.logging :as log]
+            [cljdoc.config :as cfg]
+            [cljdoc.storage.api :as storage]
+            [cljdoc.server.routes :as routes]))
+
+
+(defn sitemap [db-spec]
+  (generate-sitemap
```

Comment:
We may want to use the `generate-sitemap-and-save*` variant at some point. If we start listing sitemaps for individual doc pages then we will quickly blow past the 50k limit in a single sitemap. This probably doesn't have to be here now, I think any kind of sitemap is going to be a massive SEO improvement over having none.

Reply:
Thanks for the comments @danielcompton ! The `generate-sitemap-and-save*` function doesn't exists before version 0.3.0. (https://github.com/hashobject/sitemap/pull/4/files) However the latest on maven central is only 0.2.5. What should I do in this case?